doc/source/examples/notebooks.rst

```diff
@@ -71,6 +71,14 @@ Advanced Machine Learning Insights
    Tabular, Text and Image Model Explainers <explainer_examples>
    Outlier Detection on CIFAR10 <outlier_cifar10>
   
+Batch Processing with Seldon Core
+-----
+
+.. toctree::
+   :titlesonly:
```

Comment:
MIssing new line here. This doesn't build correctly unless you add. Plus the referenced notebook has some top-level headings (#) that need to be changed to second level (##) so as to not appear here.

Reply:
Done